LayoutTests/http/tests/site-isolation/scrolling/remove-root-frame.html

```diff
@@ -0,0 +1,30 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script src="/js-test-resources/js-test.js"></script>
+<script src="/js-test-resources/ui-helper.js"></script>
+<body>
+    <pre id="scrollingTree"></pre>
+</body>
+<script>
+description("Verifies that root frames are successfully removed from scrolling trees when removed from the DOM.");
+jsTestIsAsync = true;
+
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();            
+}
+
+window.addEventListener('load', async () => {
+    if (!window.internals)
+        return
+    frameToRemove.parentElement.removeChild(frameToRemove);
+    if (window.GCController)
+        GCController.collect();
+    document.getElementById('scrollingTree').innerText = await UIHelper.getScrollingTree();
+    setTimeout(function() {
+        if (window.testRunner)
+            testRunner.notifyDone();
+    }, 500);
```

Comment:
Why the 500ms timeout?

Reply:
This gives time for things to happen to make the call to rootFrameWasRemoved more likely to be hit by this test.  The results are the same with or without it being hit, so it's not flaky, but this additional timeout makes it so that more code is at least hit by the test in the common case.